modules/structs/issue.go

```diff
@@ -50,8 +50,9 @@ type Issue struct {
 	Ref              string     `json:"ref"`
 	Labels           []*Label   `json:"labels"`
 	Milestone        *Milestone `json:"milestone"`
-	Assignee         *User      `json:"assignee"`
-	Assignees        []*User    `json:"assignees"`
+	// deprecated
```

Comment:
Can you mention this in the description and/or title?

Reply:
@jolheiser done